bogenliga/bogenliga-application/src/main/java/de/bogenliga/application/services/v1/user/service/UserService.java

```diff
@@ -374,4 +383,22 @@ public UserDTO create(final HttpServletRequest requestWithHeader,
     }
 
 
+    @RequestMapping(value = "{id}", method = RequestMethod.DELETE)
+    @RequiresPermission(UserPermission.CAN_DELETE_SYSTEMDATEN)
+    public UserDTO deactivate(@PathVariable("id") final long id, final Principal principal) {
+        Preconditions.checkArgument(id >= 0, "Id must not be negative.");
+
+        LOG.debug("Receive 'delete' request with Id '{}'", id);
+
+        // allow value == null, the value will be ignored
+        final UserDO userDO = new UserDO();
+        userDO.setId(id);
+
+        final UserDO existingDO = userComponent.findById(id);
+
+        final UserDO userUpdatedDO = userComponent.update(userDO, !existingDO.isActive());
```

Comment:
Wenn der User schon deaktiviert ist, wird er damit wieder aktiv? Ist das so gewünscht?!
Außerdem finde ich das nicht schön und sauber gelöst! Gebt doch einfach die ID von dem User mit in die Business Logik, der schaut dann in der Datenbank ob er existiert und setzt den status auf inaktiv.
Den Weg hier ein neuen UserDO anzulegen, die ID zu setzen und sonst nix und den dann mit rein zu füllen, ist zu viel! 
Also besser: 
- schreibt auch in der userComponent eine Schnittstelle deacitvate. Die kriegt als parameter ne ID von nem User
- in der Implementierung holt ihr euch den User mit der ID aus der Datenbank und ändert den Status auf inaktiv.